### PR TITLE
add isoptional

### PR DESCRIPTION
**Pull request by ManapatWeeramongkolkul**

### CODE REVIEW

File: src/healthRecord/healthRecord.interface.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { ApiProperty } from "@nestjs/swagger"
-import { IsArray, IsNumber, IsObject, IsString } from "class-validator"
+import { IsArray, IsNumber, IsObject, IsOptional, IsString } from "class-validator"
 import { UploadProfileRequest } from "src/user/dto/user.dto"
+import { IsNull } from "typeorm"
```
Comment (Awesomeman007) [inline src/healthRecord/healthRecord.interface.ts]: Is this used anywhere? dont forget to remove unused import.

File: src/healthRecord/healthRecord.interface.ts [hunk 2]
```diff
@@ -46,6 +47,8 @@ export class AddHealthrecord {
   @IsString()
   hrName: string
   @ApiProperty({ type: UploadImage })
+  @IsOptional()
+  @IsObject()
   picture?: UploadImage
   @ApiProperty({ type: [HealthDataField] })
   @IsArray()
```
Comment (Awesomeman007) [inline src/healthRecord/healthRecord.interface.ts:52]: Just curious why there are the same classes in both dto and interface. Is there any reason?

**Review by Awesomeman007 — APPROVED** [review]

File: src/healthRecord/healthRecord.interface.ts (hunk 2 above)
Comment (ManapatWeeramongkolkul) [reply src/healthRecord/healthRecord.interface.ts:52]: I wanted to separate dto for controller and interface for service

File: src/healthRecord/healthRecord.interface.ts (hunk 1 above)
Comment (ManapatWeeramongkolkul) [reply src/healthRecord/healthRecord.interface.ts]: UploadProfileRequest is not used by anyone too. Should I remove it too?

Comment (ManapatWeeramongkolkul) [reply src/healthRecord/healthRecord.interface.ts]: Removed

**Pull request merged**